lib/src/intl_suggestors/utils.dart

```diff
@@ -145,7 +145,7 @@ String intlGetterDef(SimpleStringLiteral node, String namespace) {
   final varName = toVariableName(node.stringValue!);
   final message = intlFunctionBody(node.stringValue!, '${namespace}_$varName',
       isMultiline: node.isMultiline);
-  return '\n\t$intlFunctionPrefix get $varName => $message;';
+  return '\n  $intlFunctionPrefix get $varName => $message;';
```

Comment:
Using tabs makes the matching in the tests painful, and there doesn't seem to be a good reason to prefer it to spaces.